quintilesims/layer0
# 596: Update AWS SDK vendoring

**Background**

The vendored version of the AWS SDK is being updated to account for recent changes that have been made in the SDK. As it pertains to issue #596, ECS ContainerDefinitions have added new properties that we should support when l0 deploys are created, including the property `LinuxParameters`


**How should this be tested?**

- Create a Layer0 instance using this branch
- Create a Linux environment
- Create a Deploy using the following Dockerrun.aws.json

```
{
    "AWSEBDockerrunVersion": 2,
    "containerDefinitions": [
        {
            "name": "guestbook",
            "image": "quintilesims/guestbook",
            "portMappings": [
                {
                    "hostPort": 80,
                    "containerPort": 80
                }
            ],
            "memory": 512,
            "linuxParameters": {
                "capabilities": {
                    "add": ["SYS_PTRACE"]
                }
            }
        }
    ],
    "essential": true
}
```
- Create a service that using this Deploy in the Environment you created
- Confirm from the AWS console that the AWS Task Definition used to create the service contains the `LinuxParameters` block in ContainerDefinitions block of the JSON contents of the Task Definition. `LinuxParameters` should not be null.

- All unit tests must pass

closes #596
